relay_hub/schemas.py

```
@@ -0,0 +1,204 @@
+    expires_at: datetime | None = None
+    ack_note: str | None = None
+    ack_metadata: Dict[str, Any] = Field(default_factory=dict)
+
+    @field_validator("tags", mode="before")
+    def normalize_tags(cls, tags: List[str]) -> List[str]:
+        if not tags:
+            return []
+        normalized_tags = []
+        for tag in tags:
+            if not tag:
+                raise ValueError("tags cannot contain empty values")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)